src/interfaces/UploadOptions.ts

```diff
@@ -104,3 +104,9 @@ export interface UploadOptions {
    */
   xhr?: XMLHttpRequest
 }
+
+export interface JwtRequestOptions {
+  publicKey: string
+  expire: number
+  uploadPayload:{[key: string]: Object}
```

Comment:
Fix formatting. Type will be `{ [key: string]: Object | string | boolean }`. Initially all values are input in this form, but we convert values to `string` before sending to `signature/v2`

Reply:
dn